src/designspace.rs

```diff
@@ -53,8 +53,7 @@ pub struct Axis {
     #[serde(rename = "@default")]
     pub default: f32,
     /// Records whether this axis needs to be hidden in interfaces.
-    #[serde(default)]
-    #[serde(rename = "@hidden")]
+    #[serde(default, rename = "@hidden", skip_serializing_if = "is_false")]
```

Comment:
maybe we can just do,
```suggestion
    #[serde(default, rename = "@hidden", skip_serializing_if = "std::convert::identity")]
```

🤔

Reply:
edit: I tried, it doesn't work :(